src/icrar/leap-accelerate/core/logging.cc

```diff
@@ -51,15 +51,15 @@ namespace log
         );
 
         //set log filter
-        #ifndef NDEBUG // Release
+        #ifndef NDEBUG // DEBUG
         boost::log::core::get()->set_filter
         (
-            boost::log::trivial::severity >= boost::log::trivial::warning
+            boost::log::trivial::severity >= boost::log::trivial::info
         );
-        #else // Debug
+        #else // RELEASE
         boost::log::core::get()->set_filter
         (
-            boost::log::trivial::severity >= boost::log::trivial::info
+            boost::log::trivial::severity >= boost::log::trivial::trace
```

Comment:
I think you want `trace` in `DEBUG` and `info` on `RELEASE`?

Reply:
true